aten/src/ATen/native/native_functions.yaml

```diff
@@ -672,7 +672,7 @@
     CPU: _embedding_bag_per_sample_weights_backward_cpu
     CUDA: _embedding_bag_per_sample_weights_backward_cuda
 
-- func: empty(int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
+- func: empty(int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, MemoryFormat memory_format=contiguous_format) -> Tensor
```

Comment:
this becomes pretty awkward because contiguous doesn't make sense for sparse -- perhaps we can just change the default to None?  Where it means "contiguous" for non-sparse and "N/A" for sparse.